fluss-rpc/src/main/proto/FlussApi.proto

```diff
@@ -947,11 +949,53 @@ message PbModifyColumn {
   optional int32 column_position_type = 4; // LAST=0,FIRST=1,AFTER=3
 }
 
-
-
 message PbDescribeConfig {
   required string config_key = 1;
   optional string config_value = 2;
   required string config_source = 3;
 }
 
+message PbLakeTableSnapshotMetadata {
+  required int64 table_id = 1;
+  required int64 snapshot_id = 2;
+  required string tiered_bucket_offsets_file_path = 3;
+  optional string readable_bucket_offsets_file_path = 4;
+}
+
+message PbLakeTableSnapshotInfo {
+  optional int64 table_id = 1;
+  required int64 snapshot_id = 2;
+  repeated PbLakeTableOffsetForBucket buckets_req = 3;
+  // add table path to reduce get table_path by table id
+  optional PbTablePath table_path = 4;
+}
+
+message PbLakeTableOffsetForBucket {
+  optional int64 partition_id = 1;
+  required int32 bucket_id = 2;
+  // Deprecated: log_start_offset is no longer used. Field number 3 is reserved for protocol compatibility.
+  // optional int64 log_start_offset = 3;
+  optional int64 log_end_offset = 4;
+  // Deprecated: partition_name is no longer used. Field number 5 is reserved for protocol compatibility.
+  // optional string partition_name = 5;
+  optional int64 max_timestamp = 6;
+}
+
+message PbPrepareCommitLakeTableRespForTable {
+  optional string lake_table_bucket_offsets_path = 1;
+  optional int32 error_code = 2;
+  optional string error_message = 3;
+}
+
+message PbTableBucketOffsets {
```

Comment:
rename to `PbTableOffsets`? This makes it more clear it is a set of offsets for a table.